source/_integrations/zwave_js.markdown

```diff
@@ -20,56 +20,66 @@ ha_codeowners:
 ha_domain: zwave_js
 ---
 
-This integration allows you to control a Z-Wave network via [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/).
+This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver and it's the successor of the [previous Z-Wave integration](https://www.home-assistant.io/integrations/zwave/) in Home Assistant, which is now considered deprecated.
```

Comment:
It's the successor to both `zwave` and `ozw` right? Maybe we should call both out

Reply:
Disregard per balloob's comment above. We may want to instead mention this on the general zwave page: https://www.home-assistant.io/docs/z-wave/

Reply:
https://github.com/home-assistant/home-assistant.io/tree/current/source/_docs/z-wave

Reply:
Yeah, let's also do a separate PR for the legacy zwave docs.